.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,13 @@
 ## Brief summary of changes
 
 
-#### Testing instructions (if applicable)
+#### Documentation files changed: 
```

Comment:
This heading doesn't make sense.. the list of documentation files changed can be seen in the diff. Why are we asking people to copy/paste it into the PR description?

Reply:
I think the motivation is to ensure that pull request authors, as well as reviewers, are made aware that documentation changes should be considered when changing code. Right now it's very easy for everybody to forget to update the documentation.

It's not a perfect solution but I do feel like it could help us keep the documentation accurate.

Do you have an alternative suggestion?

Reply:
What about a checklist that include a question as `Have you updated related documentation`?

Reply:
Yeah that might be a better way to do it.

Reply:
What @PapillonMcGill said